src/KiwiLogin/__test__/KiwiLogin.test.js

```diff
@@ -1,23 +1,68 @@
 // @flow
-
 import * as React from 'react';
+import { shallow } from 'enzyme';
+
+import { PureKiwiLogin } from '../index';
 
 describe('KiwiLogin', () => {
-  /*
-  enzyme broken until new release https://github.com/airbnb/enzyme/pull/1513
-
-  const mockStore = configureStore();
-  const store = mockStore({});
-
-  const component = mount(
-    <CloseContext.Provider value={() => {}}>
-      <MemoryRouter initialEntries={[{ key: 'testKey' }]}>
-        <KiwiLogin store={store} />
-      </MemoryRouter>
-    </CloseContext.Provider>
-  );
-  */
+  const props = {
+    onLogin: () =>
+      new Promise(resolve => {
+        process.nextTick(() => resolve());
+      }),
+    history: {
+      push: jest.fn(),
+    },
+  };
+  const e = {
+    preventDefault: jest.fn(),
+  };
+
+  const wrapper = shallow(<PureKiwiLogin {...props} />);
+  it('should change state', () => {
+    const Login = wrapper.instance();
+    const state = Login.state;
+    expect(state).toEqual({ email: '', password: '', showError: false });
+
+    wrapper.find(`[type="email"]`).simulate('change', {
+      target: { name: 'email', value: 'email@kiwi.com' },
+    });
+    wrapper.find(`[type="password"]`).simulate('change', {
+      target: { name: 'password', value: 'myRealPassword' },
+    });
+
+    expect(Login.state).toMatchSnapshot();
+  });
+
+  it('should handle sign in', async () => {
+    const Login = wrapper.instance();
+    await Login.handleSignIn(e);
+    expect(e.preventDefault.mock.calls).toHaveLength(1);
+    expect(Login.props.history.push.mock.calls[0][0]).toBe('/content');
```

Comment:
`/content` ----> `routeDefinitions.CONTENT` from `/src/routeDefinitions`.

Reply:
I personally like to have stuff hardcoded in tests, so they fail if I make some changes. But if you think this is a better approach I don't mind changing it 👍

Reply:
But aren't you testing that the history was pushed **exactly** to `routeDefinitions.CONTENT`, as expected behavior from the component?

Reply:
yep, makes sense here 👍